app/utils/email_domains.txt

```diff
@@ -1,5 +1 @@
-gsa.gov
-gmail.com
-dispostable.com
-simulator.amazonses.com
-amazonses.com
\ No newline at end of file
+gov
```

Comment:
With this change, let's move this out to a spot in `config.py` and skip the file reading. We can just call the config var `GOVERNMENT_EMAIL_DOMAIN_NAMES` and call it directly.